src/main/java/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+            componentModel = "spring",
+            injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+            nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+            implementationPackage = "<PACKAGE_NAME>.impl"
+    )
+public class MapperConfig {
```

Comment:
You can use mapper config directly in BookMapper class in the @Mapper annotation 
(minor comment)